README.md

```diff
@@ -53,8 +53,7 @@ Create a new python environment and activate it. For example, with `conda`:
 >Note `flow-merge` requires `python>=3.10`
 
 ```bash
-conda env create
-conda activate flow-merge
+conda create -n flow-merge python>=3.10 && conda activate flow-merge
```

Comment:
It looks to me as if the `conda create` is a different command from the `conda env create` where it does require `-n` or `-p` like and so it's more aimed for commands such as these below and not so much to work with environment.yml files. Do you agree, if you would double check?
```shell
#Create an environment containing the package 'sqlite':
conda create -n myenv sqlite
#Create an environment (env2) as a clone of an existing environment (env1):
conda create -n env2 --clone path/to/file/env1
```
https://conda.io/projects/conda/en/latest/commands/create.html

For setting the python>=3.10, I understand that is there because in that command there isn't assumed access to an environment.yml file, but in our case I would assume that this line below would do the job of setting the right python in the created environment. (And then setup.py (later pyproject.toml) would handle defining the supported versions of python for package). Do you see any obstacles there?
```
# environment.yml
dependencies:
  - python=3.10.13
```

Reply:
To achieve similar as in the `conda create` command, in the environment.yml, we could also avoid hardcoding the minor version by saying
```
dependencies:
  - python~=3.10
```
Which will install the latest minor version of Python 3.10, so I think currently 3.10.13.

> Note that `~=3.10` is equivalent to `>=3.10,<3.11`, which means "any version greater than or equal to 3.10 and less than 3.11". This ensures that you get the latest minor version of Python 3.10, without accidentally installing a newer major version (e.g., Python 3.11).

Reply:
Resolved: talking about just installing the package using python vs creating a development environment, we decided to split the Development section into its own section.